Changes

```diff
@@ -148,6 +148,12 @@ _______________
   The syntax '-(1 [@foo])' was incorrectly parsed as '-1'.
   (Jules Aguillon, reviewed by Gabriel Scherer, report by Gabriel Scherer)
 
+- #13058: Add TSan instrumentation to caml_call_gc(), since it may raise
+  exceptions.
+  (Fabrice Buoro, Olivier Nicole, Gabriel Scherer and Miod Vallat, review
+  by TBD).
```

Comment:
I guess in this case you can remove the "review by ..." field, given that this is more of a collaborative effort with self-reviews.

Reply:
If you want this in 5.2, you should move it in the 5.2.0 section right away.

Reply:
Done.